giraffe/src/types/index.ts

```diff
@@ -286,6 +288,22 @@ export interface LineLayerConfig {
   shadeBelowOpacity?: number
 }
 
+export interface BandLayerConfig {
+  type: 'band' // do not refactor or restrict to LayerTypes.Line
+  x: string
+  y: string
+  fill: string[]
+  position?: LinePosition
+  hoverDimension?: LineHoverDimension | 'auto'
+  maxTooltipRows?: number
+  interpolation?: LineInterpolation
+  lineWidth?: number
+  lineOpacity?: number
+  colors?: string[]
+  shadeBelow?: boolean
+  shadeBelowOpacity?: number
+}
```

Comment:
```suggestion
export interface BandLayerConfig {
  type: 'band' // do not refactor or restrict to LayerTypes.Line
  x: string
  y: string
  fill: string[]
  position?: LinePosition
  hoverDimension?: LineHoverDimension | 'auto'
  maxTooltipRows?: number
  interpolation?: LineInterpolation
  lineWidth?: number
  lineOpacity?: number
  colors?: string[]
  shadeOpacity?: number
  upperColumn: string
  lowerColumn: string
  legendMessage: string
}
```

Reply:
I will be moving "legendMessage" into the top level config since that is where the other legend configurations are located.

Reply:
Also, slight change to "upperColumn" to "upperColumnName" and similarly for "lowerColumn" to "lowerColumnName"